input/postgres/relation_stats.go

```diff
@@ -20,10 +20,7 @@ locked_relids_with_parents AS (
 	UNION SELECT relid FROM locked_relids
 )
 SELECT c.oid,
-			 COALESCE(pg_catalog.pg_table_size(c.oid), 0) +
-			 COALESCE((SELECT pg_catalog.sum(pg_catalog.pg_table_size(inhrelid))
-			    FROM pg_catalog.pg_inherits
-				 WHERE inhparent = c.oid), 0) AS size_bytes,
+			 COALESCE(pg_catalog.pg_table_size(c.oid), 0) AS size_bytes,
```

Comment:
You mention this in the PR description, but I don't follow why this change is being made. Can you clarify?

Reply:
This part of the query aggregated the table sizes from child partitions so the parent table wasn't zero-sized. But it was also inaccurate because it counted TOAST data as table data, which is why if you look at a partition parent table in pganalyze today, none of them will show TOAST usage even though the child tables use TOAST.

Instead, this PR handles this in `merge_partition_sizes.go` with correct handling of TOAST data.